.ko.yaml

```diff
@@ -1,2 +1,4 @@
 # Use :nonroot base image for all containers
 defaultBaseImage: gcr.io/distroless/static:nonroot
+baseImageOverrides:
+  github.com/knative/serving/vendor/github.com/tsenart/vegeta: ubuntu:latest
```

Comment:
nice

Reply:
Use this distroless variant ;)

https://github.com/dprotaso/distroless-ubuntu/tree/master/bionic